libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -587,6 +610,8 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 			}
 		}
 
+		// FIXME: Pointers to internal library functions are supported in Solidity but internalDispatch()
+		// does not include them so using them will result in invalid code being generated.
```

Comment:
I could not block this case with `solUnimplementedAssert()` because we only know at runtime which function the pointer points at. And I couldn't do it in `endVisit(MemberAccess)` either because there I only see that the member gets accessed, but not what for (whether it's called, assigned to something or passed to a function).

In the end I decided to just leave a comment because I'll be implementing it in another PR soon anyway.

Now the question is how should I go about making that PR. Currently `internalDispatch()` just blindly generates entries for all functions from the contract that have the same signature. I suppose that adding all such functions from all libraries is not what we want, given that there's even a task to make it include only the ones that are actually ever assigned to pointers (#6788). I could take over that issue and implement it first. What do you think?

Reply:
Let's merge this first without adding library functions to internal dispatch.